### ISSUE TITLE
Add shortcuts translation table to main docs

### ISSUE THREAD

**Issue by wdfinch** [rbreaves/kinto#368]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It seems like the only documentation listing shortcut mappings + a list of the supported shortcuts is in this [issue](https://github.com/rbreaves/kinto/issues/44). Being as this is extremely useful/important to many users, I think it should be included in the main documentation and regularly updated. For example, while browser forward/back behavior seems to be working on Firefox in Linux, it still doesn't work on Chrome in Linux (while the table in this post suggests otherwise). A note regarding this could help new users become comfortable with kinto faster and save time reading through past issues.

**Comment by rbreaves** [comment]
I agree, I do need to go through that document and get it updated and have it reflected, or link to it, in the readme.

Not sure about the browser forward and back keys on chrome, I can go back and look at that a bit later and update that, I am sure it can be updated and supported even if I thought I couldn't support it previously. Xkeysnail supports hotkeys in succession so I can pretty well activate anything via a combo of alt and whatever key presses needed. I might not have known that when I switched over to it, I know Chrome has always been a lil funky.

**Comment by rbreaves** [comment]
Just to note a mention to the translation table is made in the Shortcut creation ticket, but it does need to be given more attention directly in the readme too. https://github.com/rbreaves/kinto/issues/115

**Comment by wdfinch** [comment]
Sounds good. Thank you and glad this will be added to the main readme. 

And yes it seems that there are still issues in Chrome. I spoke about this in some previous issues so I wasn't sure what the status was here. Firefox is working perfectly in Linux.

**Comment by wdfinch** [comment]
Did you want to leave this open for now or close it?

**Comment by rbreaves** [comment]
Open for now, I will try and get to it some time this week probably.

**Comment by wdfinch** [comment]
Thank you! Kinto is an amazing project and makes it so much easier for me to use/develop in Linux.
